### ISSUE TITLE
Limit posts to n characters?

### ISSUE THREAD

**Issue by ednapiranha** [bigboringsystem/bigboringsystem#192]
Posts are now more or less unlimited - should probably put a max on it and strip the rest. Not sure what the ideal count would be - thoughts? @isaacs @jasonrhodes @imakewebthings

**Comment by jasonrhodes** [comment]
Uhhh no idea but here is a stab:

Medium says most engaging posts are about 7 minutes long on average, or 1000-1600 words. 

https://blog.bufferapp.com/the-ideal-length-of-everything-online-according-to-science

5.1 characters per English word, says Wolfram Alpha. That puts "good" posts at around 5000-10,000 characters, roughly?

So put the limit on the very high end if it's a data concern, maybe 15-20,000 chars, or the low end if it's a design and editorial constraint, so maybe 5000 then?

Haha like I said, no idea. :)

**Comment by llkats** [comment]
My first reaction was to vehemently oppose any sort of limit, but I think that it's overwhelming to read BBS posts that are super long. Someone mentioned at some point that BBS seems to occupy a space somewhere between Twitter (microposts) and Medium (long-form essays). So maybe we could try a 5000 character limit and see if people chafe against it. Anyhow, constraint fosters creativity, right?

**Comment by ednapiranha** [comment]
5000 sounds good to me. we can just do a silent clip and make a note about it .. or apply a counter i guess..

**Comment by dylancwood** [comment]
I'd like to take this on. 

I have added a character limit on both the server and client side in my forked repo, but am having trouble getting the tests to run. It seems that some tests are running in parallel, though they should be running in series. As a result, the `post` global var is not being set before it is referenced in other tests. 

I'll keep digging, and raise a separate issue if needed.

**Comment by dylancwood** [comment]
Resolved my test issues (which were not caused by parallel execution), and submitted a PR #229 . I've never used BBS before, so please let me know if the character limit is what you had imagined.
